src/library/scala/collection/convert/Wrappers.scala

```diff
@@ -411,24 +411,21 @@ private[collection] trait Wrappers {
 
     override def size = underlying.size
 
```

Comment:
Since this change is logically separate, it should be in a separate Pull Request.